src/test/java/uk/gov/hmcts/reform/cosapi/controllers/CaseManagementControllerTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.hmcts.reform.cosapi.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.cosapi.edgecase.event.EventEnum;
+import uk.gov.hmcts.reform.cosapi.edgecase.model.CaseData;
+import uk.gov.hmcts.reform.cosapi.model.CaseResponse;
+import uk.gov.hmcts.reform.cosapi.services.CaseManagementService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.cosapi.util.TestFileUtil.loadJson;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+@ActiveProfiles("test")
+@SuppressWarnings("PMD")
+public class CaseManagementControllerTest {
+
+    private final String caseTestAuth = "testAuth";
+
+    @InjectMocks
+    private CaseManagementController caseManagementController;
+
+    @Mock
+    CaseManagementService caseManagementService;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
+
+    }
+
+    @Test
+    public void testC100CreateCaseData() throws Exception {
+        ObjectMapper mapper = new ObjectMapper().registerModule(new JavaTimeModule());
```

Comment:
ObjectMapper can be put under @Before method so that you do not need to create for each test case